lib/mrb/scripts/logger/flags.rb

```diff
@@ -0,0 +1,36 @@
+module Groonga
+  class Logger
+    class Flags
+      @@flags = {}
+      @@flag_names = nil
+      class << self
+        def set_flag_names(flag)
+          flag_names = @@flags.collect do |index, candidate|
+            candidate.name if candidate.flag & flag > 0
+          end
+          @@flag_names = flag_names.compact.join("|")
+        end
+      end
+
+      attr_reader :name, :flag
+      def initialize(name, flag)
+        @@flags[flag] = self
+        @name  = name
+        @flag = flag
+      end
+
+      def to_s
+        @@flags_names
```

Comment:
Why `@name` isn't used?

Reply:
`@name` is implicitly used when correcting each flag name.

Reply:
(`@name` is not enough to collect all default flags, so I've used `@flags_names` in the previous version)